Lib/gftools/tests/test_fix.py

```diff
@@ -237,3 +242,22 @@ def test_fix_vertical_metrics_typo_metrics_enabled(static_fonts):
         assert font["OS/2"].sTypoDescender == -300
         assert font["OS/2"].sTypoLineGap == 0
     _check_vertical_metrics(static_fonts)
+
+
+def test_fix_colr_v0_font(colr_font):
+    # Fix a COLR v0 font.
+    # maximum_color should not be run and GID 1 should have a blank glyph
+    from gftools.fix import fix_colr_font
+    colr_font["COLR"].version = 0
+    fixed = fix_colr_font(colr_font)
+    assert "SVG " not in fixed
+    gid1 = fixed.getGlyphOrder()[1]
+    assert fixed["glyf"][gid1].numberOfContours == 0
+
+
+def test_fix_colr_v1_font(colr_font):
+    # Fix a COLR v1 font. Only maximum_color should be run
+    from gftools.fix import fix_colr_font
+    colr_font["COLR"].version = 1
```

Comment:
you're loading a COLRv0 font (CairoPlay) but then setting its version to 1.. that's not sufficient for a valid conversion from version 0 to 1; there are additional attribtues that you're ignoring, also the fonttools API for v0 and v1 is slightly different for historical reasons (we kept backward compat for the v0 one to not break existing tools). I would suggest that in your pytest fixture you actually load a real COLRv1 test font (a small subset is fine).

Reply:
Thanks. I'll use real fonts.